tdrs-frontend/package.json

```diff
@@ -9,23 +9,23 @@
     "axios": "^0.21.1",
     "classnames": "^2.3.0",
     "connected-react-router": "^6.8.0",
-    "file-type": "^16.3.0",
+    "file-type": "^16.5.1",
```

Comment:
Yarn doesn't seem to have the same issue as pipenv for updating unrelated packages so these `^` versions are fine here.